lib/kernel/src/net_kernel.erl

```diff
@@ -2456,17 +2453,31 @@ hidden_argument() ->
             false
     end.
 
+%%%
+%%% -proto_dist command line argument
+%%%
+
+proto_dist_argument() ->
+    case init:get_argument(proto_dist) of
+        {ok, [Protos | Rest]} ->
+            case Rest of
+                [] ->
+                    ok;
+                _ ->
+                    ?LOG_WARNING("Multiple -proto_dist given to erl, using the first, ~p", [Protos])
```

Comment:
Note: the warning shows up twice, since we read the value in two places.

Reply:
I think that's fine. :-)